test/unit/media/headset.test.ts

```diff
@@ -167,6 +167,31 @@ describe('SdkHeadset', () => {
       expect(setHoldSpy).toHaveBeenCalledWith('123', false);
     });
   });
+
+  describe('listenForSessionEvents', () => {
+    it('should end currentCall on sessionEnded', () => {
+      sdkHeadset.endCurrentCall = jest.fn();
+
+      const conversationId = 'myConvoId';
+      sdk.emit('sessionEnded', { conversationId } as any, null);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9596369)
```suggestion
      sdk.emit('sessionEnded', { conversationId } as unknown, null);
```